src/components/common/adblock_detector.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { HTMLAttributes } from 'react';
+
+interface Props extends HTMLAttributes<HTMLButtonElement> {}
+
+interface State {}
+
+class AdBlockDetector extends React.Component<Props, State> {
+    constructor(props: Props) {
+        super(props);
+    }
+
+    public componentDidMount = async () => {
+        const isAddBlockDetected = await this.detectAdBlock();
+        if (isAddBlockDetected) {
+            alert('Ad-Block DETECTED, please disable it in order to use our dApp!');
+            window.location.reload();
```

Comment:
I think we should remove the reload and show a message like "We detected you are using an ad blocker. Keep in mind that this dapp may not work correctly with it enabled". @pablofullana thoughts?

Reply:
I removed the reload and changed the message, But nevertheless I'll wait the response of @pablofullana to close this conversation

Reply:
I support the no-reload approach.

About styling, I'd definitely go with a 'nicer' one. What about a pop up such as the one proposed in here: https://github.com/0xProject/0x-launch-kit-frontend/issues/118

Reply:
I'd added @gabitoesmiapodo in order to add those styles :)